lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -0,0 +1,113 @@
+var edx = edx || {};
+
+(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CohortDiscussionsView = Backbone.View.extend({
+
+        /**
+         Add/Remove the disabled attribute on given element.
+
+         Args:
+            $element (JQuery element): The element to disable/enable.
+            disable (Bool): The flag to add/remove 'disabled' attribute.
+        **/
+        setDisabled: function($element, disable) {
+            if (disable) {
+                $element.prop('disabled', 'disabled');
+            } else {
+                $element.prop('disabled', false);
+            }
+        },
+
+        /**
+         Returns the cohorted discussions list.
+
+         Args:
+            selector (HTML element): The topic element to get the ids.
```

Comment:
How about "selector (string): To select the discussion elements whose ids to return."?